include/nn/ui2d/Pane.h

```diff
@@ -41,19 +41,25 @@ class Pane {
     void RemoveChild(nn::ui2d::Pane*);
     void GetVertexPos() const;
 
-    u64 _8;
+    nn::ui2d::Pane* mParent;  // _8
     u64 _10;
     u64 _18;
     u64 _20;
     u64 _28;
-    u64 _30;
-    u64 _38;
-    u64 _40;
-    u32 _48;
-    u32 _4C;
-    u64 _50;
-    u16 _58;
-    u16 _5A;
+    u32 mPositionX;      // _30
```

Comment:
Are you sure these are not floats?

Reply:
No, they definitely are floats (best to be seen in the ctor). Changed accordingly.